firebase-sessions/firebase-sessions.gradle.kts

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+plugins {
+  id("firebase-library")
+  id("kotlin-android")
+}
+
+firebaseLibrary {
+  testLab.enabled = true
+  publishSources = true
+}
+
+android {
+  val targetSdkVersion: Int by rootProject
+  compileSdk = targetSdkVersion
+  defaultConfig {
+    minSdk = 16
+    targetSdk = targetSdkVersion
+    multiDexEnabled = true
+    testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+  }
+  compileOptions {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+  }
+  kotlinOptions { jvmTarget = "1.8" }
+  testOptions.unitTests.isIncludeAndroidResources = true
+}
+
+dependencies {
+  implementation(project(":firebase-common"))
+  implementation(project(":firebase-common:ktx"))
```

Comment:
I think you only need a dependency on firebase-common:ktx here. The other ones appear to get pulled in transitively.

Reply:
Removed